### ISSUE TITLE
New bootstrap+vue3 project 

### ISSUE THREAD

**Issue by corbinu** [bootstrap-vue/bootstrap-vue#6556]
Given https://github.com/bootstrap-vue/bootstrap-vue/issues/5196 was getting a bit muddled in its message I am creating this issue to gauge interest in both participation and features for a new Bootstrap Vue project. 

First off I want to express how much I loved this project and thank the maintainers for all their help. 

The idea would be to create a new project focused on Vue3 support. I am going to pose a few suggestions for possible features/directions as separate comments so people can easily express their thoughts with the reaction emojis. Obviously welcome any thoughts, make other suggestions people can react to, or simply add their name as interested in contributing. 

Here are my first thoughts:

- I would like to get something going quick/simple even it has fewer features as people do need to get moving forward on Bootstrap+Vue projects.
- Nuxt support should be planned but not part of the first MVP
- Given that Vue3 is out and Bootstrap 5 is eminent I would probably want to start from scratch with a project targeting those rather than a fork
- Should be structured from the start to allow many components to be easily added, however.
- I am really enjoying using Vite so I would probably lean towards a newer build stack like Vite or Snowpack rather than vue-cli
- I don't have any need for IE11 support and given that it has been dropped from Vue3 and Bootstrap 5. I think it would be best to start without it and possibly add backward compatibly with Vue2 and Bootstrap 4 later as personally, I am ok with if the end result is "well it is a bunch of shims and slow but we have IE11 support and when the time comes we can easily remove"

**Comment by xvaara** [comment]
do we want to reimplement the bootstrap js in "native vue" or just use bootstraps own js like here: [https://dev.to/codeply/using-bootstrap-5-with-vue-js-5fnp](https://dev.to/codeply/using-bootstrap-5-with-vue-js-5fnp)

There is also this: [https://github.com/superbvue/SuperBVue](https://github.com/superbvue/SuperBVue)
and [material desin bootstrap](https://github.com/mdbootstrap/mdb-vue-ui-kit)

**Comment by corbinu** [comment]
Suggestion issue: Library Support

- 🚀 Bootstrap 5 on Vue3
- 🎉 Bootstrap 4 and 5 on Vue3
- ❤️ Bootstrap 4 and 5 on Vue2

**Comment by corbinu** [comment]
Suggestion issue: Need IE11

- 👍 for "yes I need it"
- 😕 for "I might need it so lets plan for adding support long term"
- 👎 for "no I don't need it"

**Comment by corbinu** [comment]
Suggestion issue: To Fork or Not to Fork that is the question!

- 👍 Fork
- 👎 Not to Fork

**Comment by corbinu** [comment]
Suggestion issue @xvaara: How to handle JS

- ❤️ Use Bootstrap JS
- 💔 Reimplement all in VueJS

(This may depend on if we end up supporting Bootstrap 4 and want to avoid depending on jQuery)

**Comment by corbinu** [comment]
Suggestion issue: Development platform

- 🚀 vite
- ❤️ vue-cli
- 🎉 snowpack
- 😕 other

**Comment by xvaara** [comment]
It might be complex to implement both bootstrap 4 and 5, there are differences in the class names. So every component would need two versions.

**Comment by corbinu** [comment]
@xvaara Agreed! still wanting to gauge interest however as if it is not needed then problem solved if it really is we can look at at least making it easy for somebody to add it like allowing the class names to be overridden even if we target BS5 first.

Thansk for the suggestions will take a look at them tonight.

**Comment by csicky** [comment]
💔 I can't select this, it is not in the emoji selector.

**Comment by corbinu** [comment]
> 💔 I can't select this, it is not in the emoji selector.

sorry fixing!

**Comment by jackmu95** [comment]
BootstrapVue is still maintained and will have Vue 3 and Bootstrap 5 support in the future.
There will not be ETAs in the near future but it will definitely happen.

I can totally understand that some people need/want a Bootstrap 5 based Vue 3 library right now and anyone in the BootstrapVue or Vue community should feel free to create one but I think this is the wrong stage to discuss this.

**Comment by corbinu** [comment]
@jackmu95 that is great news! and obviously this was born simply out of the desire simply to get some sort of ball rolling and to stop the muddling of the message on the Vue3 issue.

Given the project is moving forward which was my preferred option might I suggest we reopen (or open a new one) and retitle this issue to community feedback for what people are looking for in Bootstrap-Vue as I do feel like all these suggestions for feedback are still valuable to the project moving forward. Clearly community members almost instantly started giving that feedback it seems like there is a real communication need here. 

I would be happy to take on managing keeping track of suggestions. Maybe a title of “Community Suggestions for version 3”?

**Comment by jackmu95** [comment]
@corbinu Sounds good 👍🏻

**Comment by tweichart** [comment]
@corbinu on the documentation side, would [Storybook](https://storybook.js.org/) be an option? I've seen it lately in quite a lot of projects, also bigger library projects with whole design systems attached. They also seem to [support Vue3 already](https://storybook.js.org/blog/storybook-vue3/). Now I haven't worked with it enough or did any of the documentation for bootstrapVue, but I just thought I might mention it as well in case this may be a good option.

**Comment by jackmu95** [comment]
@tweichart I think people like/prefer a documentation that is like the Bootstrap one.

**Comment by sudazzle** [comment]
I think new version should be vue3 and Bootstrap 5 only. I think by the time bootstrap-vue vue3 support comes in bootstrap 5 stable release should be out.

**Comment by Hiws** [comment]
> I think new version should be vue3 and Bootstrap 5 only. I think by the time bootstrap-vue vue3 support comes in bootstrap 5 stable release should be out. What branch is WIP for vue3 support?

Based on the discussion below, a stable release of Bootstrap 5 *should* be coming in the next couple of weeks.
https://github.com/twbs/bootstrap/discussions/33617
